Dockerfile

```diff
@@ -0,0 +1,124 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright 2020 (C) Olliver Schinagl <oliver@schinagl.nl>
+
+# hadolint ignore=DL3007  latest is the latest stable for alpine
+FROM registry.hub.docker.com/library/alpine:latest AS builder
+
+LABEL Maintainer="Olliver Schinagl <oliver@schinagl.nl>"
+
+WORKDIR /build
+
+COPY . /build/
+
+# hadolint ignore=DL3008  We want the latest stable versions
+RUN apk add --no-cache \
+        autoconf \
```

Comment:
Swap for `cmake` please! 😀

For running tests, also add `python3` and `check-dev`

Reply:
Hah! I used the alpine apkbuild as input; i'll switch though and add the tests for sure

Reply:
Done, though I couldn't get the tests working as part of the build sadly, and didn't think it was worth-while to add them for this use-case specifically. I can revisit this if needed however.

I looked at your preliminary github actions yaml file, and invoked the test the same way, but this is what ctest yells ...
```console
UpdateCTestConfiguration  from :/src/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/src/build/DartConfiguration.tcl
Test project /src/build
Constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
No tests were found!!!
```